src/plots/gl3d/scene.js

```diff
@@ -340,7 +341,7 @@ function Scene(options, fullLayout) {
     this.spikeOptions = createSpikeOptions(fullLayout[this.id]);
     this.container = sceneContainer;
     this.staticMode = !!options.staticPlot;
-    this.pixelRatio = options.plotGlPixelRatio || 2;
+    this.pixelRatio = this.pixelRatio || options.plotGlPixelRatio || 2;
```

Comment:
This may break graphs where users set config option `plotGlPixelRatio`:

https://github.com/plotly/plotly.js/blob/13b925cc2c946fddf78dd0201ace5f9e986de545/src/plot_api/plot_config.js#L309-L319

Orca does it here:

https://github.com/plotly/orca/blob/776d5c3f1faf7272f7eb6e1c11ba8d7fb0057734/src/component/plotly-graph/render.js#L30

using (a strange) value:

https://github.com/plotly/orca/blob/776d5c3f1faf7272f7eb6e1c11ba8d7fb0057734/src/component/plotly-graph/constants.js#L35

Reply:
Oh wait nevermind, `this.pixelRatio` isn't set by default (correct?) so `options.plotGlPixelRatio` is honoured.

Reply:
Yes I think so.

Reply:
Ok great. Can you try generating the `gl3d_*` baseline using orca with a few different `plotGlPixelRatio` values?

You may want to modify:

https://github.com/plotly/plotly.js/blob/3e15f17b7aa5399bc36655ee5a8da7b1787ca450/tasks/noci_test.sh#L31-L39

to get orca started easily and you might have to tweak the orca `src` files e.g. here:

https://github.com/plotly/orca/blob/776d5c3f1faf7272f7eb6e1c11ba8d7fb0057734/src/component/plotly-graph/constants.js#L35

or 

https://github.com/plotly/orca/blob/776d5c3f1faf7272f7eb6e1c11ba8d7fb0057734/src/component/plotly-graph/render.js#L30

to pass the different `plotGlPixelRatio` values down to plotly.js

Reply:
@etpinard Thank you so much for the links. Using orca I was able to produce gl3d_* images from this branch and compared them against the master branch. I also tweaked the `pixelRatio` of orca module and the configuration has an impact.